ui/component/autoplayCountdown/view.jsx

```diff
@@ -52,22 +57,53 @@ function AutoplayCountdown(props: Props) {
     }
   }, [navigateUrl, nextRecommendedUri, isFloating, doSetPlayingUri, doPlayUri]);
 
+  function debounce(fn, time) {
+    let timeoutId;
+    return wrapper;
+    function wrapper(...args) {
+      if (timeoutId) {
+        clearTimeout(timeoutId);
+      }
+      timeoutId = setTimeout(() => {
+        timeoutId = null;
+        fn(...args);
+      }, time);
+    }
+  }
+
+  React.useEffect(() => {
+    const handleScroll = debounce(e => {
+      const elm = document.querySelector('.autoplay-countdown');
```

Comment:
Can you extract this class to a constant at the top of this file?

Reply:
Updated.  Hopefully I understood the comment correctly.